File: test-suite/src/test/java/io/micronaut/jsonschema/test/JsonSchemaConfig.java
package io.micronaut.jsonschema.test;

import io.micronaut.jsonschema.JsonSchemaConfiguration;

/**
 * A configuration.
 */
// tag::clazz[]
@JsonSchemaConfiguration(
    baseUri = "https://example.com/schemas" // <1>
)
public interface JsonSchemaConfig {
}
// end::clazz[]

File: test-suite-kotlin/src/test/kotlin/io/micronaut/jsonschema/test/JsonSchemaConfig.kt
package io.micronaut.jsonschema.test

import io.micronaut.jsonschema.JsonSchemaConfiguration

/**
 * A configuration.
 */
//tag::clazz[]
@JsonSchemaConfiguration(baseUri = "https://example.com/schemas")
interface JsonSchemaConfig
//end::clazz[]